lib/events/test/suite.go

```diff
@@ -189,7 +188,7 @@ Outer:
 		event, ok := arr[0].(*apievents.UserLogin)
 		c.Assert(ok, check.Equals, true)
 		c.Assert(event.GetTime(), check.Equals, baseTime2)
-		c.Assert(slices.Contains(names, event.User), check.Equals, true)
+		c.Assert(apiutils.SliceContainsStr(names, event.User), check.Equals, true)
```

Comment:
I think this was meant to be `slices.Contains` from `golang.org/x/exp/slices`

Reply:
Yup, that was the intent, I got caught out by expecting we had Go 1.18 already.